pages/about.html

```diff
@@ -1,12 +1,41 @@
-<!DOCTYPE html>
+<!DOCTYPE html> 
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8"> 
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
 </head>
 <body>
-  
+  <header>
+    <h1 class="page-title">About Alie </h1>
+    <nav> 
+      <ul> 
+        <li><a class="link" href="html_index.asp">Index</a></li>
+        <li><a class="link" href="html_about.asp">About</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section>
+    <h3 class="page-subtitle"> Before Seattle </h3>
```

Comment:
These should be `h2` tags since they're the second level of headings (if you want to change the style/sizing of headings you should use a CSS rule on `h2`s):

```suggestion
    <h2 class="page-subtitle"> Before Seattle </h3>
```